File: FoodOrdering/Models/Order.cs
namespace FoodOrdering.Models;

public class Order
{
    public int Id { get; set; }
    public int TableNumber { get; set; }
    public int FoodItemId { get; set; }
    public FoodItem FoodItem { get; set; }
    public int Quantity { get; set; }
    public decimal TotalPrice { get; set; }
    public DateTimeOffset OrderDate { get; set; }
    public OrderState OrderState { get; set; }
}

File: FoodOrderingClient/src/app/models/order.ts
import {FoodItem} from './food-item';
import {OrderState} from './order-state';

export interface Order {
  id: number;
  tableNumber: number;
  foodItemId: number;
  foodItem: FoodItem;
  orderDate: Date; // using Date for DateTimeOffset
  orderState: OrderState;
}
